cmd/oras/root/file.go

```diff
@@ -70,7 +70,7 @@ func addFile(ctx context.Context, store *file.Store, name string, mediaType stri
 	if err != nil {
 		var pathErr *fs.PathError
 		if errors.As(err, &pathErr) {
-			err = fmt.Errorf("%s: no such file or directory", pathErr.Path)
+			err = fmt.Errorf("%s %s: %w", pathErr.Op, pathErr.Path, pathErr.Err)
```

Comment:
What the difference between above and `err = pathErr`?

Reply:
The returned type is not `*fs.PathError` any more? I think we can use `err = pathErr` here